lib/view/icu_non_intensivist/general_care_view.dart

```diff
@@ -11,9 +11,13 @@ class GeneralCareView extends StatelessWidget {
     return Scaffold(
       backgroundColor: AppColors.backgroundBlue,
       appBar: AppBar(
-          backgroundColor: AppColors.appBarBackground,
-          iconTheme: Styles.appBarIconTheme,
-          title: Text(title, style: Styles.appBarTextStyle)),
+        // Warning brightness interacts with SystemUiOverlayStyle
+        // See system_bars.dart comments
+        brightness: Brightness.light,
```

Comment:
do we need to hardcode this, wont it default to `.light` ?

Reply:
I'm afraid so for now. Not sure why just yet, but AppBar not following the theme properly and so without it we get the white on white status bar again. So have to do it on every AppBar for now.